examples/Arithmetic/main.roc

```diff
@@ -0,0 +1,65 @@
+app "example"
+    packages { pf: "https://github.com/roc-lang/basic-cli/releases/download/0.3.1/97mY3sUwo433-pcnEQUlMhn-sWiIf_J9bPhcAFZoqY4.tar.br" }
+    imports [
+        pf.Stdout,
+        pf.Task.{ await },
+        pf.Arg,
+    ]
+    provides [main] to pf
+
+TaskErrors : [InvalidArg, InvalidNumStr]
+
+main =
+    task =
+        args <- readArgs |> await
+
+        calculate = \(operation, result) ->
+            resultStr = Num.toStr result
+
+            "\(operation): \(resultStr)"
```

Comment:
Given that this function does no calculation, we should probably call it something else :p